### PR TITLE
Changelog 2019.03

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,57 @@
+Version 2019.03-RC (UNRELEASED, 2019-03-xx)
+  Friendica Core:
+    Update to the translation (CS, DE, EN_US, ES, PL, ZH-CN) [translation teams]
+    Update to the documentation [Aditoo17, JeroenED, m4sk1n, softmetz, tobiasd]
+    Update to the themes (frio, smoothy, quattro) [lxiter, MrPetovan. rabuzarus,  tobiasd]
+    Enhancements to the API [jasonscheng]
+    Enhancements to the Vagrand development VM [JeroenED]
```
Comment (annando) [inline CHANGELOG]: Isn't it "Vagrant"?

Comment (MrPetovan) [reply CHANGELOG]: Good catch.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+    Enhancements to the explicit and implicit mentioning in conversations [annando, MrPetovan]
+    Enhancements to the warnings in the admin panel [tobiasd]
+    Enhancements to the AP implementation [annando]
+    Enhancements to the worker process of deleting items [annando]
+    Enhancements to the testing process [nupplaphil]
+    Enhancements to the LibreJS compatibility [tobiasd]
+    Fixed several bugs with weird tagging in code blocks [lxiter]
+    Fixed a bug during updating contact entries [rabuzarus]
+    Fixed several PHP warnings and errors [annando, MrPetovan, tobiasd]
+    Fixed an issue when Friendica is installaed in a subdirectory [rabuzarus]
```
Comment (MrPetovan) [inline CHANGELOG]:
```suggestion
    Fixed an issue when Friendica is installed in a subdirectory [rabuzarus]
```

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,57 @@
...
+    Fixed several issues in the handling of the Markdown parsing from and to d* [lxiter, MrPetovan]
+    Fixed a bug in the ACL with preselected items [lxiter]
+    Fixed an issue with polling Events via AP [annando]
+    Fixed a memory issue with the JSON-LD parser [annando]
+    General Code cleaning and restructureing [annando, JeroenED, MrPetovan]
```
Comment (MrPetovan) [inline CHANGELOG]:
```suggestion
    General Code cleaning and restructuring [annando, JeroenED, MrPetovan]
```

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,57 @@
...
+    Updates to the translations (CS, DE, ES, FR, NL, PL) [translation teams]
+    Updated documentation [softmetz, tobiasd]
+    cookienotice:
+      Added new addon to display a cookie usage notice in the browser [lxiter]
+    forumdirectory:
+      Fixed a theming issue with frio [rabuzarus]
+    mailstream:
+      Fixed a curl issue [MrPetovan]
+    piwik:
+      Make it clear that Piwik and Matomo is meant [tobiasd]
```
Comment (MrPetovan) [inline CHANGELOG]:
```suggestion
      Make it clear that Piwik is now Matomo [tobiasd]
```

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,57 @@
...
+    Enhancements to the display of contacts in the profile [MrPetovan]
+    Enhancements to the handling of local links [lxiter]
+    Enhancements to the explicit and implicit mentioning in conversations [annando, MrPetovan]
+    Enhancements to the warnings in the admin panel [tobiasd]
+    Enhancements to the AP implementation [annando]
+    Enhancements to the worker process of deleting items [annando]
+    Enhancements to the testing process [nupplaphil]
+    Enhancements to the LibreJS compatibility [tobiasd]
+    Fixed several bugs with weird tagging in code blocks [lxiter]
+    Fixed a bug during updating contact entries [rabuzarus]
```
Comment (MrPetovan) [inline CHANGELOG]:
```suggestion
    Fixed a bug during contact entries update [rabuzarus]
```

File: CHANGELOG [hunk 6]
```diff
@@ -38,13 +38,20 @@ Version 2019.03-RC (UNRELEASED, 2019-03-xx)
...
     If the node does not want to publish information, nodeinfo now returns 404 instead of non-existing entries [nupplaphil]
     General Code cleaning and restructuring [annando, JeroenED, MrPetovan, nupplaphil]
     Switched logging engine to monolog [nupplaphil]
     Added plugable storage backend [fabrixxm, nupplaphil]
     Added item delivery indicator to posting header [MrPetovan]
     Added URL parameter to force a specific language [JeroenED]
     Added live preview of attachments to frio [rabuzarus]
+    Added a "friendica_author" to the API results which holds the real author [annando]
+    Added support for AP with forum postings [annando]
+    Added a summery of articles send over AP [annando]
```
Comment (annando) [inline CHANGELOG]: "summary"

File: CHANGELOG [hunk 7]
```diff
@@ -1,3 +1,92 @@
+Version 2019.03-RC (UNRELEASED, 2019-03-xx)
+  Friendica Core:
+    Update to the translation (CS, DE, EN-GB, EN-US, ES, FR, IT, PL, SV, ZH-CN) [translation teams]
+    Update to the documentation [Aditoo17, JeroenED, m4sk1n, softmetz, tobiasd]
+    Update to the themes (duepuntozero, frio, smoothy, quattro, vier) [lxiter, MrPetovan. nupplaphil, rabuzarus,  tobiasd]
```
Comment (AlfredSK) [inline CHANGELOG]: There's a point behind MrPetovan. Should be a comma.

**Pull request merged**